packages/components/playwright-ct.config.ts

```diff
@@ -0,0 +1,49 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { defineConfig, devices } from '@playwright/experimental-ct-react';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+    testDir: './src',
+    /* The base directory, relative to the config file, for snapshot files created with toMatchSnapshot and toHaveScreenshot. */
+    snapshotDir: './__snapshots__',
+    /* Maximum time one test can run for. */
+    timeout: 10 * 1000,
+    /* Run tests in files in parallel */
+    fullyParallel: true,
+    /* Fail the build on CI if you accidentally left test.only in the source code. */
+    forbidOnly: !!process.env.CI,
+    /* Retry on CI only */
+    retries: process.env.CI ? 2 : 0,
+    /* Opt out of parallel tests on CI. */
+    workers: process.env.CI ? 1 : undefined,
+    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+    reporter: 'html',
+    /* Test match pattern. See https://playwright.dev/docs/api/class-testmatch */
+    testMatch: '**/*.integ.{ts,tsx}',
+    /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+    use: {
+        /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+        trace: 'on-first-retry',
+        /* Port to use for Playwright component endpoint. */
+        ctPort: 3100,
+    },
+
+    /* Configure projects for major browsers */
+    projects: [
```

Comment:
Probably CI would need a `matrix` to parallelize the browsers

Reply:
I am testing CI behaviour until this PR is draft to check different options for executions

Reply:
@SamuelAlev do you mean you want to run all browsers also on different OS?

Windows -> (chromium, firefox, webkit)
Linux -> (chromium, firefox, webkit)
MacOS -> (chromium, firefox, webkit)

Reply:
What I meant is `matrix: ["chromium", "firefox", "safari"]` then it will execute 3 workers for the 3 tests suites

Reply:
I think it's a bit early to scale to a 3x3 matrix? I would suggest with start with one OS and enable more VMs as we begin adding tests in the next weeks?

Reply:
Sure 👍